### PR TITLE
Modify run_mojo command to take path to devtools.

### PR DESCRIPTION
**Pull request by alhaad**
Made changes as discussed.
R=@jamesr

### CODE REVIEW

**Comment by jamesr** [comment]
You don't need different flags.  --android already controls whether this script tries to run directly or use devtools

**Comment by alhaad** [comment]
PTAL.

/cc @abarth

File: lib/src/commands/run_mojo.dart [hunk 1]
```diff
@@ -27,7 +27,7 @@ class RunMojoCommand extends Command {
     argParser.addFlag('mojo-release', negatable: false, help: 'Use Release build of mojo (default)');
 
     argParser.addOption('app', defaultsTo: 'app.flx');
-    argParser.addOption('mojo-path', help: 'Path to directory containing mojo_shell and services');
+    argParser.addOption('mojo-path', help: 'Path to directory containing mojo_shell and services for Linux and to mojo devtools from Android.');
```
Comment (jamesr) [inline lib/src/commands/run_mojo.dart]: leave a TODO(jamesr) in a comment here for the mojo_tools package to figure this out

**Comment by jamesr** [comment]
Please squash your commits together

**Comment by jamesr** [comment]
lgtm

**Pull request merged**